paas/install_client.go

```diff
@@ -176,36 +176,32 @@ func (c *InstallClient) showInstallConfiguration(opts *kubernetes.InstallationOp
 func (c *InstallClient) fillInMissingSystemDomain(domain *kubernetes.InstallationOption) error {
 	if domain.Value.(string) == "" {
 		ip := ""
-		if !c.kubeClient.GetPlatform().HasLoadBalancer() {
-			ips := c.kubeClient.GetPlatform().ExternalIPs()
-			if len(ips) > 0 {
-				ip = ips[0]
-			}
-		} else {
-			s := c.ui.Progressf("Waiting for LoadBalancer IP on traefik service.")
-			defer s.Stop()
-			timeout := time.After(2 * time.Minute)
-			ticker := time.Tick(1 * time.Second)
-		Exit:
-			for {
-				select {
-				case <-timeout:
+		s := c.ui.Progressf("Waiting for LoadBalancer IP on traefik service.")
+		defer s.Stop()
+		timeout := time.After(2 * time.Minute)
+		ticker := time.Tick(1 * time.Second)
+	Exit:
+		for {
+			select {
+			case <-timeout:
+				return errors.New("Timed out waiting for LoadBalancer IP on traefik service.\n" +
+					"Ensure your kubernetes platform has the ability to provision LoadBalancer IP address.\n\n" +
+					"Follow these steps to enable this ability\n" +
+					"https://github.com/SUSE/carrier/blob/main/docs/install.md#provision-of-external-ip-for-loadbalancer-service-type-in-kubernetes")
```

Comment:
Although it's easier for the user, you should probably avoid the anchor link (everything after `#`) because it will change if we simply reword the title. Just point to the file. Also put a TODO to update this link when we get a proper domain for carrier docs (e.g. docs.carrier.suse.dev or something like that)

Reply:
Done.